Changes

```diff
@@ -1,5 +1,16 @@
 Release history for Zonemaster component Zonemaster-GUI 
 
+v3.0.1 2019-01-31 (pre-release version)
+ 
+ - Status
+   - This is a pre-release version not fully tested on all supported
+     OS's and Perl versions. This version will not be available on
```

Comment:
No need to speak about Perl. The GUI is only dependent on the version of the browser (but the Linux version of Firefox is not the same than the Windows version).

Reply:
You are right. Good point. Let me fix here. It is confusing.

For GUI we should specify what browsers and OS that we test. Let us do that for next version.